docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,708 @@
+```
+
+For screencast there is more configuration needed, it's described [here](https://github.com/jellyfish-dev/react-native-client-sdk#ios).
+
+We also suggest setting background mode to `audio` so that the app doesn't
+disconnect when it's in the background:
+
+```xml title=ios/jellyfish-dashboard/Info.plist
+<key>UIBackgroundModes</key>
+<array>
+  <string>audio</string>
+</array>
+```
+
+  </TabItem>
+   <TabItem value="expo-bare" label="Expo Bare workflow"> 
+   
+   You have two options here. You can follow configuration instructions for
+React Native (Expo Bare workflow is a React Native project after all) or if
+you're using `expo prebuild` command to set up native code you can add our Expo
+plugin. Just add it to `app.json`:
+
+```json title:app.json
+{
+  "expo": {
+    "name": "example",
+    //...
+    "plugins": ["@jellyfish-dev/react-native-membrane-webrtc"]
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Add components library
+
+For your convenience, we've prepared a library with nice-looking components
+useful for following this tutorial. Feel free to use standard React Native components or your own components though!
+
+```bash
+npx expo install @expo/vector-icons expo-barcode-scanner expo-font @expo-google-fonts/noto-sans @jellyfish-dev/react-native-jellyfish-components @react-navigation/native-stack
+```
+
+**_You'll also need to install [Reanimated library (3.3.0)](https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started) and [React Navigation (6.1.7)](https://reactnavigation.org/)_**
+
+:::tip
+Run `pod install` in the /ios directory to install the new pods
+:::
+
+## Connecting to the server and joining the room
+
+Our app will consist of two screens. The first one allows the user to type, paste or
+scan the peer token and connect to the room. The second screen shows room
+participants with their video tracks.
+
+### Screens
+
+For managing screens I've used React Navigation library, you're free though to
+use another one. Our app will contain two screens: Connect screen and Room
+screen:
+
+```tsx title="/App.tsx"
+import React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createNativeStackNavigator } from "@react-navigation/native-stack";
+import ConnectScreen from "./screens/Connect";
+import RoomScreen from "./screens/Room";
+
+const Stack = createNativeStackNavigator();
+
+function App(): JSX.Element {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name="Connect" component={ConnectScreen} />
+        <Stack.Screen name="Room" component={RoomScreen} />
+      </Stack.Navigator>
+    </NavigationContainer>
+  );
+}
+
+export default App;
+```
+
+### Connect screen
+
+The UI of the Connect screen consists of a simple text input and a few buttons.
+The flow for this screen is simple: the user either copies the peer token from the
+dashboard or scans it with a QR code scanner and presses Connect button. The QR
```

Comment:
Could we start a new sentence in a new line? This make it easier to track changes and add suggestions